### PR TITLE
Use system zstd

### PR DESCRIPTION
**Pull request by dbast**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

closes https://github.com/conda-forge/zstandard-feedstock/issues/43

### CODE REVIEW

**Comment by dbast** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,21 +10,24 @@ source:
...
     - python
     - pip
-    - cffi >=1.8
+    - cffi >=1.11
+    - zstd
   run:
     - python
-    - cffi >=1.8
+    - cffi >=1.11
+    - zstd
```
Comment (xhochy) [inline recipe/meta.yaml:30]: This will be added by `run_exports` automatically.
```suggestion
```

**Comment by dbast** [comment]
@conda-forge-admin, please rerender

**Comment by dbast** [comment]
hm: error: option --with-system-zstd not recognized

**Pull request merged**